borg/helpers.py

```diff
@@ -879,7 +929,13 @@ def remove_surrogates(s, errors='replace'):
 def make_path_safe(path):
     """Make path safe by making it relative and local
     """
-    return _safe_re.sub('', path) or '.'
+    if sys.platform != 'win32':
+        return _safe_re.sub('', path) or '.'
+    else:
+        tail = path
+        if path[0:2] == '//' or path[0:2] == '\\\\' or path[1] == ':':
+            drive, tail = os.path.splitdrive(path)
```

Comment:
I don't think storing drive letter is meaningful and as said paths with drive letter are absolute.